equinix/resource_network_device.go

```diff
@@ -539,6 +600,7 @@ func createNetworkDeviceSchema() map[string]*schema.Schema {
 					networkDeviceSchemaNames["VendorConfiguration"]: {
 						Type:     schema.TypeMap,
 						Optional: true,
+						Computed: true,
```

Comment:
Let's add a list of the map keys to `networkDeviceDescriptions["VendorConfiguration"]` since users may be confused that these map keys are not identical to the new node schema names.

We will address that inconsistency in the future, but that will be a breaking change.  As discussed, we will defer that until after this PR and feature are released.